editoast/Dockerfile

```diff
@@ -46,13 +46,14 @@ COPY --from=planner /editoast/recipe.json recipe.json
 COPY --from=planner /editoast/editoast_derive/ editoast_derive
 ENV RUSTFLAGS="-C link-arg=-fuse-ld=mold"
 ARG CARGO_PROFILE=release
+ARG CARGO_FEATURES
 RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=/editoast/target \
-    cargo chef cook --profile ${CARGO_PROFILE} --recipe-path recipe.json
+    cargo chef cook --features ${CARGO_FEATURES} --profile ${CARGO_PROFILE} --recipe-path recipe.json
```

Comment:
Hm. `${}` is docker-style expansion, so I'm worried this gets expanded to `cook --features  --profile debug` for local builds. Does this work fine still?

Reply:
Oh, right... why didn't I thought about that (thank you peer review 😄). It probably doesn't work indeed. Should be fix in the last push.